schema/bancornetwork/view_add_smart_token.sql

```diff
@@ -9,26 +25,26 @@ SELECT r."exchange_token",
        q.contract_address,
        q.evt_tx_hash AS tx_hash,
        q.evt_block_time AS block_time
-FROM bancornetwork."ConverterRegistry_evt_SmartTokenAdded" q
+FROM smart_token_added q
 LEFT JOIN
-  (SELECT DISTINCT ON (s."_smartToken") s."_convertibleToken" AS "exchange_token",
+  (SELECT DISTINCT ON (s.smart_token) s.convertible_token AS "exchange_token",
                       t2.symbol AS "exchange_token_symbol",
                       t2.decimals AS "exchange_token_decimals",
-                      p."_convertibleToken" AS "base_token",
+                      p.convertible_token AS "base_token",
                       p.symbol AS "base_token_symbol",
                       p.decimals AS "base_token_decimals",
-                      s."_smartToken" AS smart_token
-   FROM bancornetwork."ConverterRegistry_evt_ConvertibleTokenAdded" s
+                      s.smart_token
+   FROM bancornetwork.view_add_convertible_token s
    INNER JOIN
      (SELECT *
       FROM bancornetwork."ConverterRegistry_evt_ConvertibleTokenAdded"
```

Comment:
Heads up that this table does not exist in Dune, they're all versioned with `v1`, `v2` ... etc

Reply:
https://github.com/duneanalytics/abstractions/pull/83/commits/44456eecab599f6949c8d42a7833dcc1e9c65be4 takes care of this.